examples/autocomplete.amp.html

```diff
@@ -0,0 +1,118 @@
+<!doctype html>
```

Comment:
plz run through validator to be sure valid then add to `test-example-validation`